oem/firstvoices/ios/FVShared/FVShared.swift

```diff
@@ -24,12 +24,13 @@ struct FVConstants {
   static let legacy_kFVKeyboardFilenameKey = "FVKeyboardFilename"
 
   // Keyboard files and paths
-  static let keyboardsPath = "Keyboards/files"
+  static let keyboardsPath = "Keyboards"
+  static let keyboardsPackage = "fv_all"
+  static let keyboardsPackageExt = "kmp"
   static let keyboardsCSVPath = "Keyboards"
   static let keyboardsCSVName = "keyboards"
   static let keyboardsCSVType = "csv"
```

Comment:
Does the ios app still use the csv file?

Reply:
Yes.  It defines the regions used for the following screen:

![Screen Shot 2021-02-11 at 1 01 57 PM](https://user-images.githubusercontent.com/25213402/107606650-564a2e00-6c69-11eb-9d4e-6be2af36e624.png)